docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -24,37 +44,11 @@ To learn more about Expectations, see [Expectation](../../terms/expectation.md).
 
 4. Click **New Expectation**.
 
-5. Select one of the following Expectation types:
-
-    - [expect_column_max_to_be_between](https://greatexpectations.io/expectations/expect_column_max_to_be_between) - Expect the column maximum to be between a minimum and a maximum value.
-
-    - [expect_column_mean_to_be_between](https://greatexpectations.io/expectations/expect_column_mean_to_be_between) - Expect the column mean to be between a minimum and a maximum value (inclusive).
-
-    - [expect_column_median_to_be_between](https://greatexpectations.io/expectations/expect_column_median_to_be_between) - Expect the column median to be between a minimum and a maximum value.
-
-    - [expect_column_min_to_be_between](https://greatexpectations.io/expectations/expect_column_min_to_be_between) - Expect the column minimum to be between a minimum value and a maximum value.
-
-    - [expect_column_values_to_be_in_set](https://greatexpectations.io/expectations/expect_column_values_to_be_in_set) - Expect each column value to be in a given set.
-
-    - [expect_column_values_to_be_in_type_list](https://greatexpectations.io/expectations/expect_column_values_to_be_in_type_list) - Expect a column to contain values from a specified type list.
-
-    - [expect_column_values_to_be_null](https://greatexpectations.io/expectations/expect_column_values_to_be_null) - Expect the column values to be null.
-
-    - [expect_column_values_to_be_of_type](https://greatexpectations.io/expectations/expect_column_values_to_be_of_type) - Expect a column to contain values of a specified data type.
-
-    - [expect_column_values_to_be_unique](https://greatexpectations.io/expectations/expect_column_values_to_be_unique) - Expect each column value to be unique.
-
-    - [expect_column_values_to_not_be_null](https://greatexpectations.io/expectations/expect_column_values_to_not_be_null) - Expect the column values to not be null.
-
-    - [expect_table_columns_to_match_ordered_list](https://greatexpectations.io/expectations/expect_table_columns_to_match_ordered_list) - Expect the columns to exactly match a specified list.
-
-    - [expect_table_row_count_to_be_between](https://greatexpectations.io/expectations/expect_table_row_count_to_be_between) - Expect the number of rows to be between two values.
-
-    - [expect_table_row_count_to_equal](https://greatexpectations.io/expectations/expect_table_row_count_to_equal) - Expect the number of rows to equal a value.
+5. Select an Expectation type. See [Available Expectation types](#available-expectation-types).
 
     If you prefer to work in a code editor, or you want to configure an Expectation from the [Expectations Gallery](https://greatexpectations.io/expectations/) that isn't listed, click the **JSON Editor** tab and define your Expectation parameters in the code pane.
 
-6. Enter the column name and then complete the optional fields.
+6. Enter the column name in the **Create Expectation** pane and then complete the optional fields.
```

Comment:
It looks like not all of the Expectations that are made available for this method of creating Expectations require a column name.  The create Expectation option lets the user select any of the available Expectations, and the `expect_table_row_*`  Expectations don't require a column name.  It might be better for this step to simply be: "Fill in the fields for the Expectation's parameters." -- or something like that.

Reply:
@Rachel-Reverie, good catch! I have updated the procedures.